Sources_v3/Controllers/ChannelController.swift

```diff
@@ -150,6 +179,9 @@ public class ChannelControllerGeneric<ExtraData: ExtraDataTypes>: Controller, De
     /// - Parameter completion: Called when the controller has finished fetching remote data. If the data fetching fails, the `error`
     /// variable contains more details about the problem.
     public func startUpdating(_ completion: ((_ error: Error?) -> Void)? = nil) {
+        setChannelObserver()
+        setMessagesObserver()
```

Comment:
I'd suggest calling these in the init. The @Cached wrapper will crash if someone tries to access it without these being set. The closures are evaluated lazily so it doesn't matter.